pkg/jsonnet/implementation/implementation.go

```diff
@@ -0,0 +1,16 @@
+package implementation
+
+import (
+	"fmt"
+
+	"github.com/grafana/tanka/pkg/jsonnet/implementation/goimpl"
+	"github.com/grafana/tanka/pkg/jsonnet/implementation/types"
+)
+
+func Get(name string) (types.JsonnetImplementation, error) {
```

Comment:
Could we have a docstring for this one or a slightly longer name? Took me a second to realise what "implementation" meant - I was like: but isn't all of the code the implementation‽

Reply:
The implementation is the jsonnet code, yes. But here, that's exactly what we do. We chose which implementation we want. I added some docstrings